build.sbt

```diff
@@ -182,7 +182,7 @@ lazy val `algebra-imdb` = project
     `pms-core`,
   )
 
-lazy val `algebra-movie` = project
+lazy val `algebra-movie` = (project in file("algebras/movie"))
```

Comment:
create a method that creates an "algebras" project, and one that creates a "utils" project, and one that creates a "services" project.

Reply:
e.g.:
```
lazy val `algebra-movie` = algebraProject("movie")
 .settings(...)
 /// as before
```